facade/logstash.go

```diff
@@ -215,7 +215,7 @@ func getFilters(configs []servicedefinition.LogConfig, filterDefs map[string]str
 		for _, filtName := range config.Filters {
 			//do not write duplicate types, logstash doesn't handle this
 			if !utils.StringInSlice(config.Type, *typeFilter) {
-				filters += fmt.Sprintf("\n  if [type] == \"%s\" {\n    %s\n  }\n",
+				filters += fmt.Sprintf("\n  if \"%s\" in [file] {\n    %s\n  }\n",
```

Comment:
We have this issue because in logstash.conf `if` statements like `if [type] == "zenmodeler"` don't work. We don't have multiline error parsing issue if I delete all those ^ conditions or if I use conditions like:
`if [file] == "/opt/zenoss/log/zenmodeler.log"`
`if "zenmodeler" in "/opt/zenoss/log/zenmodeler.log"`
`if "zenmodeler" in [file]`
This is my current logstash.conf:
https://gist.github.com/olst/971858d0dcd5c03cbf5d3765f797d3fd
issue:
https://jira.zenoss.com/secure/attachment/42765/42765_zenmodelerUI.png
fix:
https://jira.zenoss.com/secure/attachment/42928/42928_bbc.png